bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/AdaptiveImageServlet.java

```diff
@@ -297,7 +298,9 @@ private void transformAndStreamAsset(SlingHttpServletResponse response, ValueMap
         String extension = mimeTypeService.getExtension(imageType);
         if ("gif".equalsIgnoreCase(extension) || "svg".equalsIgnoreCase(extension)) {
             LOGGER.debug("GIF or SVG asset detected; will render the original rendition.");
-            stream(response, asset.getOriginal().getStream(), imageType, imageName);
+            try (InputStream inputStream = asset.getOriginal().getStream()) {
```

Comment:
This leads to false positive in spotbugs with Java 11 (https://github.com/spotbugs/spotbugs/issues/1338).